pan/functions.pan

```diff
@@ -92,9 +83,8 @@ function npush = {
     n = dict();
     index = 0;
     while (index < ARGC) {
-        if (!is_string(ARGV[index]))
-        error(format("key is not a string: %s", ARGV[index]));
-        n[ARGV[index]] = ARGV[index + 1];
+        if (!is_string(ARGV[index])) error(format("key is not a string: %s", ARGV[index]));
```

Comment:
not sure what pan compiler version we should be targetting thee days, but as of 10.7, you can remove the format

Reply:
IMO, it is a little bit too early to require 10.7 in the core templates...